app/src/main/java/io/github/gmathi/novellibrary/NovelLibraryApplication.kt

```diff
@@ -159,9 +159,12 @@ class NovelLibraryApplication : MultiDexApplication() {
     }
 
     override fun attachBaseContext(base: Context) {
-        super.attachBaseContext(base)
-        MultiDex.install(this)
```

Comment:
We don't need this? `MultiDex.install(this)`

Reply:
this is what super.attachBaseContext looks like:
``` java
public class MultiDexApplication extends Application {
  @Override
  protected void attachBaseContext(Context base) {
    super.attachBaseContext(base);
    MultiDex.install(this);
  }
}
```
Personally, I have never used Multidex, just normal Application

Reply:
> If you do override the Application class, change it to extend MultiDexApplication (if possible) as follows:
```kotlin
class MyApplication : MultiDexApplication() {...}
```
If you don't extend MultiDexApplication then:
```kotlin
class MyApplication : SomeOtherApplication() {
    override fun attachBaseContext(base: Context) {
        super.attachBaseContext(base)
        MultiDex.install(this)
    }
}
```
so no need for MultiDex.install(this)

Reply:
Ah! missed that detail!